test/PracticeWebApi.Test/UnitTest1.cs

```diff
@@ -1,10 +1,13 @@
-namespace PracticeWebApi.Test;
+using System;
+using System.Diagnostics;
 
-public class UnitTest1
-{
-    [Fact]
-    public void Test1()
-    {
+namespace PracticeWebApi.Test {
 
+    public class UnitTest1 {
+        [Fact]
+        public void Test1() {
+            int val = 1;
+            Debug.Assert(val != 1, " Value should not be 1.");
```

Comment:
Good stuff!
The important thing is that for testing we want to use testing liblaries, like the one standard for dot net. https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.assert.areequal
Can be replaced with something like
```
int val = 1;
Assert.AreEqual(val, 1);
```

Reply:
When I used Assert.AreEqual() it threw some errors maybe because we used xUnit to create this testing project, while the first method is from the NUnit testing library, as I see here: https://learn.microsoft.com/en-us/visualstudio/test/getting-started-with-unit-testing?view=vs-2022&tabs=dotnet%2Cxunit. So I used the Assert.Equal/.NotEqual method.

Reply:
You're absolutely correct! I missed that you're using xUnit. It should be just `Assert.Equal` indeed 👍